packages/fds-components/src/style/interactive/buttonGroup.css

```diff
@@ -1,5 +1,11 @@
 :root {
   --button-group-timeout: 250ms;
+  --button-group-focus-color: rgba(0, 51, 102, 0.32);
+  --button-group-hover-color: #DEE5EB;
+  --button-group-pressed-color: #C2CEDA;
+  --button-group-active-color: #DBE2E9;
+  --button-group-disabled-border-color: #DBE2E9;
+  --border-radius-default: 4px;
```

Comment:
I've added a bunch of variables around some of the "questionable" category of values (colors, radius). The radius will eventually live in `fds-dictionary`. The colors will not, as it's been communicated that these are specific to the ButtonGroup.

I wonder if it's "hurtful" to do stuff like this, as if the user is inspecting the active variables in the browser, they may  be confused as to what's "global" variables, vs variables specific for a style sheet (and whether there should be some convention that differentiates "global" variables from "local" variables.

Reply:
Anything specified on `:root` will be global. If you're concerned about devs inspecting, there are two things that I think make this okay:

1. `fds-dictionary` is the one source of truth for standard CSS vars, not the inspector
2. In chrome and firefox, custom properties are split by files the properties are defined in, not by selector:

![Screen Shot 2019-07-23 at 5 19 11 PM](https://user-images.githubusercontent.com/231252/61748441-3644d800-ad6e-11e9-8256-aa13091d421a.png)

Reply:
Right. I do feel like this cuts both ways, because I also wouldn't want developers to overuse these file level variables. I feel like the variables I used are smart (only things that fit into fds-dictionary theme... colors, transition timeout, etc), but a develop could see that and start coding in arbitrary spacing, flexbox stuff, etc.